src/nupic/encoders/adaptivescalar.py

```diff
@@ -195,22 +195,9 @@ def topDownCompute(self, encoded):
     return super(AdaptiveScalarEncoder, self).topDownCompute(encoded)
 
 
-  def dump(self):
-    """
-    Prints details about current state to stdout.
-    """
-    print "AdaptiveScalarEncoder:"
-    print "  min: %f" % self.minval
-    print "  max: %f" % self.maxval
-    print "  w:   %d" % self.w
-    print "  n:   %d" % self.n
-    print "  resolution: %f" % self.resolution
-    print "  radius:     %f" % self.radius
-    print "  periodic: %s" % self.periodic
-    print "  nInternal: %d" % self.nInternal
-    print "  rangeInternal: %f" % self.rangeInternal
-    print "  padding: %d" % self.padding
-
+  def __str__(self):
+    return "AdaptiveScalarEncoder\n  min: %f\n  max: %f\n  w: %d\n  n: %d\n  resolution: %f\n  radius: %f\n  periodic: %f\n  nInternal: %f\n  rangeInternal: %f\n  padding: %d" \
+            % (self.minval, self.maxval, self.w, self.n, self.resolution, self.radius, self.periodic, self.nInternal, self.rangeInternal, self.padding)
```

Comment:
Please use the max line length of 79 chars specified in [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).